core/src/main/java/com/klaytn/caver/transaction/type/AccessTuple.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2022 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.transaction.type;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.web3j.rlp.RlpList;
+import org.web3j.rlp.RlpString;
+import org.web3j.rlp.RlpType;
+import org.web3j.utils.Numeric;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * AccessTuple is a class representing an element of EIP-2930 access list.
+ */
+@JsonDeserialize(using = AccessTuple.AccessTupleDeserializer.class)
+public class AccessTuple {
```

Comment:
This seems like different with specifications.
Can you check the spec?
And if you want to change, then please let me know

Reply:
@jimni1222 

I applied the changes by [other PR](https://github.com/klaytn/caver-java/pull/291/commits/fef6b05e34a82faeacc1b12a313526d5290c2adc).
* Implemented `AccessList` and wrapper.
* Implemented TransactionUtils

Please take a look.